Doc/glossary.rst

```diff
@@ -1104,7 +1104,7 @@ Glossary
       The :class:`collections.abc.Sequence` abstract base class
       defines a much richer interface that goes beyond just
       :meth:`~object.__getitem__` and :meth:`~object.__len__`, adding
-      :meth:`count`, :meth:`index`, :meth:`~object.__contains__`, and
+      :meth:`!count`, :meth:`!index`, :meth:`~object.__contains__`, and
       :meth:`~object.__reversed__`. Types that implement this expanded
```

Comment:
I think you're correct that we don't need a link for each one of these methods, but perhaps you could work in a reference to https://docs.python.org/3/library/stdtypes.html#common-sequence-operations somewhere here. The existing link to `collections.abc.Sequence` doesn't actually provide any documentation for any of these methods, unfortunately.

Reply:
Would the reference to `collections.abc.Sequence` at the beginning of the paragraph be sufficient? Or maybe that should be a link to more pedestrian `common sequence operations`? When someone looks up a term in the glossary, I'm not sure we should expect them to be conversant in abstract base classes...

Reply:
I think it would be useful to link to both `collections.abc.Sequence` and the table on common sequence operations. The former link is useful because we need to distinguish between the glossary term "sequence" and the ABC `Sequence`. The latter link is useful because the docs for `collections.abc.Sequence` don't actually document any of the ABC's abstract methods (and nor do they directly link to documentation for any of its abstract methods)

Reply:
So, the bare minimum is `__getitem__` and `__len__`, but there's more to sequences than just that. The glossary entry mentions a few others. I could graft on a link to [Common Sequence Operations](https://docs.python.org/3/library/stdtypes.html#common-sequence-operations), but it seems cleaner to leave `__getitem__` and `__len__` there, but remove the other methods which are mentioned and instead provide that external link. Something like:

```
The :class:`collections.abc.Sequence` abstract base class
defines a much richer interface that goes beyond just
:meth:`~object.__getitem__` and :meth:`~object.__len__`.
`Common Set Operations</library/stdtypes.html#common-sequence-operations>`_
provides a list common to most sequence types. Types which implement this expanded
interface can be registered explicitly using
:func:`~abc.ABCMeta.register`.
```

Does that seem better to you?

(I'm not sure how to actually link to a completely different document within the set. That's just a stab.)

Reply:
I'm getting pretty confused about where commentary is. I just pushed the above change (after figuring out the `:ref:...` syntax). LMK.